### PR TITLE
Updated the extension list

### LINKED ISSUE #3544: [Docs RFC] Update the extensions page
* Update https://docs.timescale.com/use-timescale/latest/extensions/ for all the latest extensions. 
* Add a column to say if it is enabled by default
* If possible, link to a page or doc saying how to use the extension
* Remember to add links to pgai and pgvectorscale

### PR DESCRIPTION
**Pull request by atovpeko**

### CODE REVIEW

**Review by billy-the-fish — APPROVED** [review]
LGTM. Need technical approval.

File: use-timescale/extensions/index.md
Comment (minkimipt) [inline use-timescale/extensions/index.md]: it's called `ai` in the extension list

File: use-timescale/extensions/index.md
Comment (minkimipt) [inline use-timescale/extensions/index.md]: it's called `vector` in the extensions list

File: use-timescale/extensions/index.md
Comment (minkimipt) [inline use-timescale/extensions/index.md]: it's called `vectorscale` in the extension list

**Review by alexeyklyukin — COMMENTED** [review]
Awesome, thanks! A few comments in the review.

File: use-timescale/extensions/index.md
Comment (alexeyklyukin) [inline use-timescale/extensions/index.md]: Can we rephrase "enabled for service capability" to something more user-friendly, i.e. preinstalled?

**Comment by minkimipt** [comment]
There are also a few extensions that are enabled from the start, and that are included into the whitelist, but not available in the UI:
```
pglogical
postgres_fdw
wal2json
```

pglogical, wal2json - related to logical replication feature. Logical replication is supported on Timesclae cloud, but for regular tables, not hypertables. I'll defer to @ramonguiu to decide how and whether we want to document that at all. Support of logical replication was added for one customer and we have a way of enabling logical replication in the backend, but users can't enable it in a self served way.
More about it in this [design doc](https://timescale.slab.com/posts/logical-replication-for-pg-tables-33mgxpkq)

postgres_fdw - enabled in user instances from the start and allows them to configure FDW to other tables. We may want to mention that this extension is enabled for customer instances, but I think we need a separate docs page describing how to use the FDW feature in the cloud.

Here is everything that is configured in extwlist for reference
https://github.com/timescale/timescaledb-operator/blob/d783ad154b67de3b7865dda0af569bbb40209dfc/charts/timescaledb/values.yaml#L91

File: use-timescale/extensions/index.md
Comment (alexeyklyukin) [inline use-timescale/extensions/index.md]: 👍🏻 nice (it's v16 max, and the link points to the specific version).

File: use-timescale/extensions/index.md
Comment (alexeyklyukin) [inline use-timescale/extensions/index.md]:
```suggestion
[bool_plper]: https://www.postgresql.org/docs/current/plperl-funcs.html
```

**Comment by alexeyklyukin** [comment]
Would it be convenient to the user if we group them by [PostgreSQL built-in extensions](https://www.postgresql.org/docs/current/contrib.html), "Timescale Provided" and "Third Party", instead of a single alphabetically sorted list?  This may help locate extensions that are missing (i.e. new built-ins in PostgreSQL versions); a user can still use the browser search if they need to check the presence of the name.

File: use-timescale/extensions/index.md
Comment (atovpeko) [reply use-timescale/extensions/index.md]: I know, but it's called pgai in docs, so I am using this name for consistency.

File: use-timescale/extensions/index.md
Comment (atovpeko) [reply use-timescale/extensions/index.md]: See comment above about ai

File: use-timescale/extensions/index.md
Comment (atovpeko) [reply use-timescale/extensions/index.md]: See comment above about ai

File: use-timescale/extensions/index.md
Comment (atovpeko) [reply use-timescale/extensions/index.md]: @billy-the-fish ?

**Comment by atovpeko** [comment]
> Would it be convenient to the user if we group them by [PostgreSQL built-in extensions](https://www.postgresql.org/docs/current/contrib.html), "Timescale Provided" and "Third Party", instead of a single alphabetically sorted list? This may help locate extensions that are missing (i.e. new built-ins in PostgreSQL versions); a user can still use the browser search if they need to check the presence of the name.

I think that would be awesome. @billy-the-fish ?

File: use-timescale/extensions/index.md
Comment (billy-the-fish) [reply use-timescale/extensions/index.md]: Hi @alexeyklyukin, happy to change the words, but we do need something to express what is happening. The fact that an extension is in the list means that it is preinstalled. Perhaps if we change this to say simply Enabled?

Comment (alexeyklyukin) [reply use-timescale/extensions/index.md]: @billy-the-fish thanks, indeed, "Precreated" would likely be a better term, or perhaps "Enabled by default"?

Comment (billy-the-fish) [reply use-timescale/extensions/index.md]: Enabled by default

**Comment by billy-the-fish** [comment]
> > Would it be convenient to the user if we group them by [PostgreSQL built-in extensions](https://www.postgresql.org/docs/current/contrib.html), "Timescale Provided" and "Third Party", instead of a single alphabetically sorted list? This may help locate extensions that are missing (i.e. new built-ins in PostgreSQL versions); a user can still use the browser search if they need to check the presence of the name.
> 
> I think that would be awesome. @billy-the-fish ?

Totally. Agreed.

File: use-timescale/extensions/index.md
Comment (alexeyklyukin) [inline use-timescale/extensions/index.md]: Those 3 (bool_plperl, jsonb_plperl and old_snapshot) are also postgres built-ins.

File: use-timescale/extensions/index.md
Comment (alexeyklyukin) [inline use-timescale/extensions/index.md]: plperl and plpgsql are also postgres built-ins

File: use-timescale/extensions/index.md
Comment (alexeyklyukin) [inline use-timescale/extensions/index.md:136]: this link seems wrong, it's also a built-in:
https://www.postgresql.org/docs/current/pgcrypto.html

Comment (atovpeko) [reply use-timescale/extensions/index.md:136]: It is listed in the built-in table, we are just linking to our own doc about it

File: use-timescale/extensions/index.md
Comment (alexeyklyukin) [inline use-timescale/extensions/index.md]:
```suggestion
[refint]: https://www.postgresql.org/docs/current/contrib-spi.html
```

**Review by alexeyklyukin — APPROVED** [review]
Looks great, thank you!

**Comment by billy-the-fish** [comment]
Yay, please publish @atovpeko .

**Pull request merged**
